superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/ColumnSelect.tsx

```diff
@@ -85,7 +86,13 @@ export function ColumnSelect({
     }
     if (datasetId != null) {
       cachedSupersetGet({
-        endpoint: `/api/v1/dataset/${datasetId}`,
+        endpoint: `/api/v1/dataset/${datasetId}?q=${rison.encode({
+          columns: [
+            'columns.column_name',
+            'columns.is_dttm',
+            'columns.type_generic',
```

Comment:
I'm not 100 % sure on this (more like 90 %), but I feel like `is_dttm` is no longer necessary, and `type_generic === GenericDataType.TEMPORAL` should always be true for any col that has `is_dttm === true`, and is the preferred way of checking for temporal columns in the frontend. So if we want to reduce payload size, this is kind of a low hanging fruit.

Reply:
@villebro thanks for the review. If your hypothesis is correct then it seems like a follow up PR would be to deprecate the `is_dttm` column everywhere.